be/src/storage/rowset/beta_rowset_writer.cpp

```diff
@@ -144,6 +145,29 @@ StatusOr<RowsetSharedPtr> BetaRowsetWriter::build() {
     RETURN_IF_ERROR(
             RowsetFactory::create_rowset(_context.tablet_schema, _context.rowset_path_prefix, _rowset_meta, &rowset));
     _already_built = true;
+
+    // if rowset is partial rowset, build rowset_update_state in advance to reduce cost time of apply
```

Comment:
This pr seems to construct parial_update_state not get the old default column.
The content is not consistent with the title.

Reply:
> This pr seems to construct parial_update_state not get the old default column. The content is not consistent with the title.

The purpose of getting old/default value is to construct partial_update_state, changed the title to make it easier to understand